runtime/parsing_functions.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "runtime/kphp_core.h"
+#include <regex>
+#include <iostream>
+#include <fstream>
+#include <string>
+
+/*
+ * Helper functions
+ */
+
+string clearQuotes(const string &str);
+
+string clearSingleQuotes(const string &str);
+
+string clearDoubleQuotes(const string &str);
+
+string clearExtraSpaces(const string &str);
+
+string tolower(const string &str);
+
+string trim(const string &str);
+
+/*
+ * INI parsing functions
+ */
+
+const int INI_SCANNER_NORMAL = 0;
+const int INI_SCANNER_RAW = 1;
+const int INI_SCANNER_TYPED = 2;
+
+string get_ini_section(const string &ini_entry);
+
+string get_ini_var(const string &ini_entry);
+
+string get_ini_val(const string &ini_entry);
+
+bool cast_str_to_bool(const string &ini_var);
```

Comment:
You don't need to expose this functions to header, because you use them only in one translation unit. They should be private for your `parsing_functions.cpp` file

Reply:
Okay, I am understand. But, how I will be test private functions?